tests/packages/link-library-in-subproject/meson.build

```diff
@@ -0,0 +1,17 @@
+# SPDX-FileCopyrightText: 2022 The meson-python developers
+#
+# SPDX-License-Identifier: MIT
+
+project(
+    'link-library-in-subproject',
+    'c',
+    version: '1.0.0',
+    meson_version: '>=1.2.3',
```

Comment:
The requirement for this Meson version should probably be documented here https://mesonbuild.com/meson-python/reference/meson-compatibility.html#meson-compatibility I completely forgot about it

Reply:
Our tests say that the minimum Meson version for `--skip-subprojects` to work is 1.2.0

Reply:
Documentation note added in #700